macros/dbt_utils/schema_cleanup/drop_schemas_by_prefixes.sql

```diff
@@ -6,9 +6,9 @@
         {# Fetch all schemas that use the current prefix #}
         {% do log('Fetching schemas for ' + prefix + '...', info=True) %}
         {% set schemas_table %}
-            select name
-            from sys.schemas
-            where name LIKE '{{prefix}}%'
+            select schema_name
+            from "{{ target.database }}"."information_schema"."schemata"
```

Comment:
What table is `schemata` ?

Reply:
it returns all schema that exist on that catalog.